lua/luacrun.d.tl

```diff
@@ -0,0 +1,149 @@
+--[[
+This file is part of crun, but licensed under the MIT License.
```

Comment:
could we double license it?  So we don't have to worry about too many licenses

Reply:
Yes, I also think a dual license would be better.